src/components/common/FileUploadPreview.vue

```diff
@@ -68,23 +71,20 @@ export default class FileUploadPreview extends Mixins(DocumentMixin) {
   async mounted (): Promise<void> {
     if (this.inputFile) {
       this.fileUpload = this.inputFile
+      // wait for v-file-input to settle
       await this.$nextTick()
-      this.isFileValid(this.fileUpload)
     }
-  }
-
-  async fileChange (file): Promise<void> {
-    await this.emitFileSelected(file)
+    await this.computeEmitFileValidity(this.fileUpload)
   }
 
   // Note: the validation is done this way as opposed to being all defined in the validation rules(fileUploadRules)
   //  above because Vuetify does not support async validation out of the box.  This was needed to work
   //  around this limitation.  vuelidate does support this but this would mean introducing a component that
   //  is using a different validation approach or updating all components to use vuelidate.  Have decided
   //  to do this for the time being.
-  private async validateFileInput (file): Promise<boolean> {
+  private async validateFileInput (file: File): Promise<boolean> {
     this.customErrorMessages = []
-    const isValid = this.$refs.fileUploadInput.validate() as boolean
+    const isValid = this.$refs.form.validate()
```

Comment:
As on the other page, the Vuetify v-file-input component needs to be wrapped in a form to trigger its validation. (Otherwise, this change is just a variable rename.)